### DISCUSSION TITLE
Is there a way to get the command “cat /dev/location > /dev/null &“ to run on startup?

### DISCUSSION THREAD

**Discussion by aveng3rr** [ish-app/ish#1392] [Q&A]
I have attempted to write an init script for open-rc, but I’ve failed to make any progress. It would be helpful if the instructions could be added to the tutorial. 

https://github.com/ish-app/ish/wiki/Running-in-background

**Comment by Arcerion-Dev** [comment]
You could create a profile file (`nano ~/.profile`).
All you have to do is make a file named .profile with the commands you want to run on separate lines, then it will run every time you launch the app. You could just put your command on the first line and save your file.
